src/tools/inner_node.rs

```diff
@@ -90,9 +91,13 @@ impl InnerNode {
         connector.set_verify(SslVerifyMode::NONE); // we might remove it once the keypair is solid
         let connector = connector.build();
 
+        let node = Node::new(config.pea2pea_config.clone());
+        node.start_listening()
+            .await
+            .expect("unable to start listening");
```

Comment:
```suggestion
```
this should happen after the protocols are enabled, even if it needs to be explicit in every test